src/activitypub/fediverse-bridge.ts

```diff
@@ -65,12 +65,15 @@ export class FediverseBridge {
                 content: post.content,
                 summary: null,
                 published: Temporal.Now.instant(),
-                attachments: post.imageUrl
-                    ? [
-                          new Image({
-                              url: post.imageUrl,
-                          }),
-                      ]
+                attachments: post.attachments
```

Comment:
I know you've not changed this, but Is `attachments` the correct key for the Fedify object? I noticed in the cucumber test you checked `attachment` (singular)

Reply:
Yes `attachments` is the correct key. Tested that. But if there is a single attachment, fedify internally changes it to `attachment` (singular). Example [here](https://browser.pub/https%3A%2F%2Fprincivershwal.ghost.io%2F.ghost%2Factivitypub%2Fnote%2F5df81770-176f-4357-9784-657685ce232e). This is why I am using attachment (singular) in tests.